### PR TITLE
Core & Internals: Propagation of requested_at to history table Fix #74

### LINKED ISSUE #74: requested_at not properly filled in request_history table
_Migrated from JIRA [RUCIO-2819](https://its.cern.ch/jira/browse/RUCIO-2819)_

### PR DESCRIPTION
**Pull request by jwackito**
Core & Internals: Propagation of requested_at to history table Fix #74

### CODE REVIEW

File: etc/sql/oracle/schema.sql [hunk 1]
```diff
@@ -1050,10 +1050,12 @@ CREATE INDEX "REQUESTS_RULEID_IDX" ON "REQUESTS" ("RULE_ID") COMPRESS 1 ONLINE t
 
 CREATE INDEX "REQUESTS_EXTERNALID_UQ" ON "REQUESTS" ("EXTERNAL_ID") ONLINE tablespace ATLAS_RUCIO_TRANSIENT_DATA01;
 
+ALTER TABLE REQUESTS ADD requested_at DATE;
 ALTER TABLE REQUESTS ADD submitted_at DATE;
 ALTER TABLE REQUESTS ADD started_at DATE;
 ALTER TABLE REQUESTS ADD transferred_at  DATE;
 
+ALTER TABLE REQUESTS_HISTORY ADD requested_at DATE;
```
Comment (vingar) [inline etc/sql/oracle/schema.sql:1058]: The table definition should be updated directly imo. I don't know why it was like that actually

Comment (bari12) [reply etc/sql/oracle/schema.sql:1058]: We have a complete mixture of that at the moment. I will go through it at some point and compile everything to table signatures and remove the alter statements. But in the meanwhile, just add it to the signature as well please.

Comment (jwackito) [reply etc/sql/oracle/schema.sql:1058]: I volunteer to do the clean up in the schema.sql

Comment (bari12) [reply etc/sql/oracle/schema.sql:1058]: Great! Thanks JJ; I will create a ticket.

**Pull request closed without merging**